merkle/merkle_path.go

```diff
@@ -130,7 +130,12 @@ func pathFromNodeToRootAtSnapshot(node int64, level int, snapshot int64, maxBitL
 			// some nodes to be overwritten. We have versioned tree nodes so this isn't necessary,
 			// we won't see any hashes written since the snapshot point. However we do have to account
 			// for missing levels in the tree.
-			drop := level - subtreeDepth(snapshot, level-1)
```

Comment:
You should probably get someone who understands the Merkle tree stuff (i.e. not me) to check this...